### PR TITLE
add verbose logs to E2E for debugging purposes

### PR DESCRIPTION
**Pull request by thiagowfx**

### CODE REVIEW

**Comment by thiagowfx** [comment]
@sadym-chromium As requested, here's the PR that adds verbose logging to E2E to help debug the flakiness in E2E (ubuntu-latest, headless).

Sample failure: https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/4861063256/jobs/8665646389

```
tests/test_browsing_context.py ..node:internal/errors:705
  const ex = new Error(msg);
             ^

Error: socket hang up
    at connResetException (node:internal/errors:705:14)
    at Socket.socketOnEnd (node:_http_client:[51](https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/4861063256/jobs/8665646389#step:10:52)8:23)
    at Socket.emit (node:events:[52](https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/4861063256/jobs/8665646389#step:10:53)5:35)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  code: 'ECONNRESET'
}
❌ [server-headless] Service exited unexpectedly


!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! KeyboardInterrupt !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/pathlib.py:240: KeyboardInterrupt
(to show a full traceback on KeyboardInterrupt use --full-trace)
```

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -51,6 +51,8 @@ jobs:
         # For verbose logging, set `DEBUG: 'bidiMapper:mapperDebug:*'` and `VERBOSE: true`.
         env:
           BROWSER_BIN: ${{ env.chromium_binary }}
+          DEBUG: 'bidiMapper:mapperDebug:*'
```
Comment (thiagowfx) [inline .github/workflows/e2e.yml:54]: You can also do `DEBUG: '*'`, but it's too noisy IMHO.

**Comment by thiagowfx** [comment]
Ironically, the first run passed. Will re-trigger to try to make it fail.

**Comment by thiagowfx** [comment]
Great, now it's failing: https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/4862921009/jobs/8674522969?pr=669

**Comment by sadym-chromium** [comment]
I added the full logs. Maybe it can give us extra hints on what is happening

**Comment by sadym-chromium** [comment]
It looks like a race condition during closing/launching browser. In the example above the process `12005` should not be killed:
```
.  bidiServer:internal Tue May 02 2023 19:40:26 GMT+0000 (Coordinated Universal Time) Peer ::1 disconnected. +222ms
  puppeteer:browsers:launcher Trying to kill 11940 +207ms
  puppeteer:browsers:launcher Browser process 11940 exists +1ms
  bidiServer:internal new WS request received: / +11ms
  bidiServer:SEND ▸ {"id":568,"result":{"type":"success","result":{"type":"undefined"},"realm":"-808600436653280652.5308073926180371153"}} +13ms
  puppeteer:browsers:launcher Launching /home/runner/work/chromium-bidi/chromium-bidi/chrome/linux-1138546/chrome-linux/chrome --headless --hide-scrollbars --mute-audio --disable-component-update --disable-popup-blocking --enable-automation --no-default-browser-check --no-first-run --password-store=basic --remote-debugging-port=9222 --use-mock-keychain --user-data-dir=/tmp/web-driver-bidi-server-Ph159W about:blank { detached: true, env: undefined, stdio: [ 'pipe', 'ignore', 'pipe' ] } +11ms
  puppeteer:browsers:launcher Launched 12005 +3ms
  puppeteer:browsers:launcher Browser process 11940 onExit +6ms
  bidiMapper:internal Establishing session with cdpUrl:  ws://127.0.0.1:9222/devtools/browser/18db609e-32b5-4feb-8975-16cde9ce9dd8 +148ms
  puppeteer:browsers:launcher Trying to kill 12005 +594ms
  puppeteer:browsers:launcher Browser process 12005 exists +2ms
```

**Comment by thiagowfx** [comment]
@OrKoN and I were trying to debug this today. Please see https://github.com/GoogleChromeLabs/chromium-bidi/pull/675

**Pull request closed without merging**